lib/base-chips/field_type.hashed-text.js

```diff
@@ -0,0 +1,44 @@
+var Sealious = require('../main.js');
+var Promise = require("bluebird");
+var crypto = require("crypto");
+
+var field_type_hashed_text = new Sealious.ChipTypes.FieldType({
+	name: "hashed-text",
+	extends: "text",
+	get_description: function(context, params){
+		return "Hash text with chosen algorithm (default md5)";
+	},
+	is_proper_value: function(accept, reject, context, params, new_value){
+		var pattern_array = [];
+		if (params instanceof Object){
```

Comment:
We're using `if(params !== undefined)` everywhere else, why did you use `instanceof` here?